build.cake

```diff
@@ -69,22 +90,46 @@ Task("BuildDocs")
 .IsDependentOn("Build")
 .Does(() =>
 {
-    DocFxMetadata("./docs/docfx.json");
-    DocFxBuild("./docs/docfx.json");  
+    FilePath artifactLocation = File("./src/Okta.Sdk/bin/Release/netstandard1.3/Okta.Sdk.dll");
+    DocFxMetadata(new DocFxMetadataSettings
+    {
+        OutputPath = MakeAbsolute(Directory("./docs/api/")),
+        Projects = new[] { artifactLocation }
+    });
+
+    DocFxBuild("./docs/docfx.json");
+    // Outputs to docs/_site
 });
 
-Task("CopyDocsToVersionedDirectories")
-.IsDependentOn("BuildDocs")
+Task("CloneExistingDocs")
 .Does(() =>
 {
-    Console.WriteLine("Copying docs to docs/temp/latest");
+    var tempDir = "./docs/temp";
 
-    if (DirectoryExists("./docs/temp/latest"))
+    if (DirectoryExists(tempDir))
     {
-        DeleteDirectory("./docs/temp/latest", recursive: true);
+        // Some git files are read-only, so recursively remove any attributes:
+        SetFileAttributes(GetFiles(tempDir + "/**/*.*"), System.IO.FileAttributes.Normal);
+        
+        DeleteDirectory(tempDir, recursive: true);
     }
-    
-    EnsureDirectoryExists("./docs/temp");
+
+    GitClone("https://github.com/okta/okta-sdk-dotnet.git",
+            tempDir,
+            new GitCloneSettings
+            {
+                BranchName = "gh-pages",
```

Comment:
The build would definitely break 😄 this shouldn't happen, since I already created the `gh-pages` branch. If that gets deleted, we have bigger problems.